src/luos.c

```diff
@@ -22,8 +27,9 @@ revision_t luos_version = {.Major = 1, .Minor = 3, .Build = 0};
 package_t package_table[MAX_CONTAINER_NUMBER];
 uint16_t package_number = 0;
 container_t container_table[MAX_CONTAINER_NUMBER];
-uint16_t container_number;
+uint16_t container_number = 0;
 volatile routing_table_t *routing_table_pt;
+node_state_t node_state = NODE_INIT;
```

Comment:
This function seems to be only used on the Luos_Run function, why not use it as a static variable into this function instead?

Reply:
luos_SetState() use it too, I would prefer to let it as a global variable.

Reply:
yes but luos_SetState is only used one time on Luos_Run so, no need to create a function for a unic use. Furthermore, Luos_Run already uses this variable directly.
you should remove Luos_SetSstate to be able to make this variable static on Luos_Run.

Reply:
Ok, I made the changes. I prefered to keep the SetState function to improve readability but the code is more concise in your way.